_models/models.js

```diff
@@ -123,4 +124,9 @@ exports.deleteSelectedComment = (comment_id) => {
                 }
                 return rows[0]
             })
+}
+
+exports.retrieveEndpoints = () =>{ 
+    return fs.readFile('endpoints.json','utf-8')
+        
 }
```

Comment:
You can do this without a model by requiring in the endpoints.json in the controller. Using fs readFile like this also works though!